src/backend/efiling-api/src/main/java/ca/bc/gov/open/jag/efilingapi/submission/SubmissionApiDelegateImpl.java

```diff
@@ -430,4 +434,35 @@ private ResponseEntity storeDocuments(SubmissionKey submissionKey, List<Multipar
         return ResponseEntity.ok(new UploadSubmissionDocumentsResponse().submissionId(submissionKey.getSubmissionId()).received(new BigDecimal(files.size())));
     }
 
+    @Override
+    @RolesAllowed("efiling-user")
```

Comment:
do we need a role like early adopters to roll this out in prod?

Reply:
No